content/src/view.js

```diff
@@ -2208,44 +2208,15 @@ function changeEditorText(paneState, text) {
   if (text.length && text.charAt(text.length - 1) != '\n') {
     text += '\n';
   }
-  paneState.changeHandler.suppressChange = true;
-  var saved = {}, editor = paneState.editor, session = editor.session;
-  saved.selection = session.selection.toJSON()
-  saved.atend = session.selection.getCursor().row >= session.getLength() - 1;
-  saved.folds = session.getAllFolds().map(function(fold) {
```

Comment:
@davidbau is it still required to keep records of selections and scrolled points even after changing the language. And would that be practical.